hasher-matcher-actioner/hmalib/models.py

```diff
@@ -73,6 +81,7 @@ class SignalMetadataBase(DynamoDBItem):
     signal_source: str
     signal_hash: str  # duplicated field with PDQMatchRecord having both for now to help with debuging/testing
     tags: t.List[str] = field(default_factory=list)
+    pending_opinion_change: PendingOpinionChange = PendingOpinionChange.NONE
```

Comment:
I think an explicit NONE comes with a few small value adds and would like to keep it at least for now. It's something that can be explicitly checked against in js. Additionally it means for development purposes it is easy to reset because you can request an update to none (instead of needing a custom conditional or endpoint to do so).